tests/unit/RecipientsTest.php

```diff
@@ -0,0 +1,171 @@
+                    new Response(200, [], self::jsonMock('RecipientMock'))
+                ])
+            ]
+        ];
+    }
+
+    public function balanceProvider()
+    {
+        return [
+            [
+                new MockHandler([
+                    new Response(200, [], self::jsonMock('BalanceMock'))
+                ])
+            ]
+        ];
+    }
+
+    /**
+     * @dataProvider recipientProvider
+     */
+    public function testRecipientCreate($mock)
+    {
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->recipients()->create([
+            "transfer_interval" => "weekly",
+            "transfer_day" => 5,
+            "transfer_enabled" => true,
+            "automatic_anticipation_enabled" => true,
+            "anticipatable_volume_percentage" => 85,
+            "bank_account_id" => 4841
+        ]);
+
+        $this->assertEquals(Endpoint::POST, self::getRequestMethod($requestsContainer));
+        $this->assertEquals('/1/recipients', self::getRequestUri($requestsContainer));
+        $this->assertEquals($response->getArrayCopy(), json_decode(self::jsonMock('RecipientMock'), true));
+    }
+
+    public function testRecipientList()
+    {
+        $mock = new MockHandler([
+            new Response(200, [], self::jsonMock('RecipientListMock'))
+        ]);
+
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->recipients()->getList();
+
+        $this->assertEquals(Endpoint::GET, self::getRequestMethod($requestsContainer));
+        $this->assertEquals('/1/recipients', self::getRequestUri($requestsContainer));
+        $this->assertEquals($response->getArrayCopy(), json_decode(self::jsonMock('RecipientListMock'), true));
+    }
+
+    /**
+     * @dataProvider recipientProvider
+     */
+    public function testRecipientFind($mock)
+    {
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->recipients()->get(['id' => 1]);
+
+        $this->assertEquals(Endpoint::GET, self::getRequestMethod($requestsContainer));
+        $this->assertEquals('/1/recipients/1', self::getRequestUri($requestsContainer));
+        $this->assertEquals($response->getArrayCopy(), json_decode(self::jsonMock('RecipientMock'), true));
+    }
+
+    /**
+     * @dataProvider recipientProvider
+     */
+    public function testRecipientUpdate($mock)
+    {
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->recipients()->update([
```

Comment:
It's good practice to use single quotes when there is no need for string interpolation :wink: